blocks/header/header.js

```diff
@@ -139,6 +157,8 @@ export default async function decorate(block) {
     decorateIcons(nav);
     const navWrapper = document.createElement('div');
     navWrapper.className = 'nav-wrapper';
+    // Add the logo element before the nav elements which will stack vertically
+    navWrapper.append(buildLogo());
```

Comment:
Instead of bringing the logo in via the nav document in google drive I decided to add it to the page dynamically given that it needed to go into a separate div element to the left of the two menu rows.  If that seems like a bad idea let me know and I'll revisit it!